2024/azureskane/data.json

```diff
@@ -9,30 +9,32 @@
     ]
   },
   "CommunityInfo": {
-    "Name": "Azure Skåne",
-    "Country": "Sweden",
-    "Url": "https://www.meetup.com/Azure-Skane/",
-    "SessionizeApiId": "",
-    "SessionizeEventName": "",
-    "Files": {
-      "EventPage": "index.md",
-      "SponsorsPage": "sponsors.md",
-      "LogoFile": {
-        "Description": "Azure Skåne Logo",
-        "File": "azureskane.png"
-      }
-    },
-    "SamApp": {
-      "AdditionalHosts": []
-    },
     "Locations": [
       {
+        "Longitude": 13.00184398064394,
+        "Address": null,
         "Name": "Malmö",
         "StartDate": "2024-04-19",
-        "EndDate": "2024-04-19",
         "Latitude": 55.61126802005422,
-        "Longitude": 13.00184398064394
+        "EndDate": "2024-04-19"
       }
-    ]
+    ],
+    "Files": {
+      "LogoFile": {
+        "Description": null,
+        "File": null
```

Comment:
Missing the transfer of the logo description and file name here @saeid-adz

Reply:
https://github.com/globalazure/communities/pull/444/commits/e6759063fd67f50ad722018aab800a927763e7c2